docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -456,14 +456,14 @@ RCT_EXPORT_VIEW_PROPERTY(text, NSString)
 @end
 ```
 
-This file is the manager for the Fabric Component. The manager objects are used by the React Native runtime to register the modules, the properties and the methods so that they are available to the JavaScript side.
+This file is the manager for the Fabric Component. React Native runtime uses the manager objects to register the modules, properties and methods to make them available to the JavaScript side.
```

Comment:
```suggestion
This file is the manager for the Fabric Component. React Native runtime uses manager objects to register the modules, properties and methods to make them available to the JavaScript side.
```
Is it right that with plurals we should not use the article? 
If not, feel free to ignore this comment